src/main/java/org/rocstreaming/roctoolkit/RocLogger.java

```diff
@@ -16,6 +16,7 @@ public class RocLogger {
 
     static {
         RocLibrary.loadLibrary();
+        setLevel(RocLogLevel.DEBUG); // set debug level and rely on jul to filter messages.
         setHandlerNative(DEFAULT_HANDLER);
```

Comment:
nit: we can rename setHandlerNative to setHandler

Reply:
done